pom.xml

```diff
@@ -155,6 +161,13 @@
 			<version>2.2.9</version>
 			<scope>test</scope>
 		</dependency>
+
+		<!-- TRUESKILL DEPENDENCIES -->
+		<dependency>
+			<groupId>de.davidbilge</groupId>
+			<artifactId>jskill</artifactId>
+			<version>1.0-SNAPSHOT</version>
```

Comment:
Oh, I see. I think it's best to depend on fixed versions, so if you can release it yourself, that would be great.